ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3290,6 +3290,9 @@ spanning_tree:
     bpdufilter_default: < true | false >
     bpduguard_default: < true | false >
   mode: < mstp | rstp | rapid-pvst | none >
+  bpduguard_rate_limit:
+    default: < true | false >
+    count: < integer >
```

Comment:
Please add some clarification explaining what is this <integer>.

Reply:
Sure